config/di-web.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Definitions\Reference;
+use Yiisoft\Request\Body\BadRequestAction;
+use Yiisoft\Request\Body\RequestBodyParser;
+
+return [
+    RequestBodyParser::class => [
+        '__construct()' => [
+            'badRequestAction' => Reference::to(BadRequestAction::class),
```

Comment:
We need to make this naming consistent. I like the `action`. @vjik what do you think?